src/core/constants.js

```diff
@@ -95,6 +95,14 @@ export const TRACEID_VRAM_VB = 'VRAM.Vb';
  */
 export const TRACEID_VRAM_IB = 'VRAM.Ib';
 
+/**
+ * Logs the vram use by the storage buffers.
```

Comment:
Does this need a `(WebGPU only)`?

Reply:
I would report 0 on other platforms, so it works.